addons/l10n_it_edi_sdicoop/models/account_invoice.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import fields, models
+
+
+_logger = logging.getLogger(__name__)
+
+DEFAULT_FACTUR_ITALIAN_DATE_FORMAT = '%Y-%m-%d'
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_transaction = fields.Char()
+    l10n_it_edi_attachment_id = fields.Many2one('ir.attachment')
+
+    def send_pec_mail(self):
```

Comment:
why public method?

Reply:
It's an override. I added that in the comment.